OJPP.js

```diff
@@ -139,7 +139,7 @@ async function zahtevaj_zamudo(busId) {
     return zamude;
 }
 
-async function izpisi_zamudo(gumb, busId) {
+async function izpisi_zamudo(gumb, busId, stPostaj = 5) {
```

Comment:
I would suggest that you always create all <li> elements, but make those > 5 with a "hidden" class. 
Then, at the bottom of the list, add a button "Show more/all" which removes the "hidden" class from the elements.

Therefore, the delay calculation would only be executed once.